build.gradle

```diff
@@ -224,6 +224,21 @@ test {
         maxFailures = 30
         maxRetries = 5
     }
+    jacoco {
+        excludes = [
```

Comment:
Why these dependencies need to be excluded from Jacoco?

Reply:
In JDK17 these class files are not instrumentable (I don't have the background just dealt with the error), had to remove them to prevent test failures.

None of these exclusions are classes that we implement in our code, and thus need to be instrumented